include/openssl/bio.h.in

```diff
@@ -427,6 +430,7 @@ struct bio_dgram_sctp_prinfo {
 #  define BIO_get_conn_port(b)          ((const char *)BIO_ptr_ctrl(b,BIO_C_GET_CONNECT,1))
 #  define BIO_get_conn_address(b)       ((const BIO_ADDR *)BIO_ptr_ctrl(b,BIO_C_GET_CONNECT,2))
 #  define BIO_get_conn_ip_family(b)     BIO_ctrl(b,BIO_C_GET_CONNECT,3,NULL)
+#  define BIO_get_conn_mode(b,n)        BIO_ctrl(b,BIO_C_GET_CONNECT,4,NULL)
```

Comment:
Not a fan of the magic numbers

Reply:
Look at all the other code around it... :/

Reply:
Yeah ok. I realised you were just following the pattern of the existing code. It's still horrible though - but another PR.